newsfragments/8464.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fix for Linux snap's users where devices are removed when the snap is removed (``snap remove parsec``).
+The fix include set ``PARSEC_BASE_CONFIG_DIR`` to ``$HOME/.config`` and have a script that copy devices present in snap user data to the new location.
```

Comment:
The migration script would be executed on `install` or `refresh`, but not on `remove` is that so?
I would rephrase it a little bit:
```suggestion
Fix for lost devices when parsec snap is removed: devices are now stored in ``$HOME/.config`` and are independent from snap. Existing devices are migrated.
```